docs/tutorial/part-one/index.md

```diff
@@ -226,35 +228,35 @@ export default () => (
 
 ![Passing data to header](07-pass-data-header.png)
 
-You should now see your “About Gatsby” header text again!
+अब आपको अपना “About Gatsby”  हेडर टेक्स्ट फिर से दिखना चाहिए!
 
-### What are “props”?
+### “प्रॉप्स” क्या है?
 
-Earlier you defined React components as reusable pieces of code describing a UI. To make these reusable pieces dynamic you need to be able to supply them with different data. You do that with input called “props". Props are (appropriately enough) properties supplied to React components.
+इससे पहले आपने रिक्ट कौम्पोनॅन्ट को कोड के रियूज़बल पीसस के रूप में परिभाषित करते हुए UI के रूप मे वर्णन किया था। इन रियूज़बल पीसस को गतिशील बनाने के लिए आपको अलग-अलग डेटा के साथ सप्लाई करना होगा। आप ऐसा करते हैं "प्रॉप्स"  इनपुट के साथ। प्रॉप्स(उचित रूप से पर्याप्त) दरअसल प्रॉपर्टीस है जिसे रिक्ट कौम्पोनॅन्ट को सप्लाई करते है।
```

Comment:
`साथ। प्रॉप्स(उचित रूप से पर्याप्त) दरअसल प्रॉपर्टीस` => `साथ। Props (उचित रूप से पर्याप्त) प्रॉपर्टीस`

Reply:
missing space between `Props` and `(उचित रूप से पर्याप्त)`